bus.py

```diff
@@ -0,0 +1,20 @@
+from vehicles import Vehicles
+
+class Bus (Vehicles):
+    
+    def __init__(self, model, price, manufactureYear, maxSpeed, passengerCount):
```

Comment:
Please add [type annotations](https://docs.python.org/3/library/typing.html) to your ```__init__``` methods. With the current implementation, I'm able to do things like ```my_bus.price = "string, but I think it was supposed to be an integer, no?"``` or ```my_bus.model = 1```